src/components/Tool/FixCommitmentInfo/GetInvalidStatuses.graphql

```diff
@@ -1,16 +1,35 @@
-query GetInvalidStatuses($accountListId: ID!) {
+query InvalidStatuses($accountListId: ID!, $after: String) {
   contacts(
     accountListId: $accountListId
     contactsFilter: { statusValid: false }
-    first: 100
+    after: $after
+    first: 50
   ) {
     nodes {
       id
       name
+      avatar
       status
       pledgeAmount
       pledgeCurrency
       pledgeFrequency
```

Comment:
So on MPDX angular, it automatically loads suggested changes to the contact's commitment info. You can try looking on the old MPDX to see how it works. But basically you can pull in `suggestedChanges` from the graphql. Then if it has a suggested status or amount or frequency, have that filled in instead of the current status, amount or frequency.
```suggestion
suggestedChanges
```

Reply:
Oh that's strange that you get that string. Did you try requesting the whole object like: 
```
suggestedChanges {
    status 
    pledgeAmount
    }
```
@dr-bizz what do you think about this?

Reply:
Hmm, then yeah that might be something Andrew or Shelby need to fix. Let's see what the good doctor @dr-bizz thinks.

Reply:
The `suggestedChanges` field is returning Ruby code which isn't right. This needs to be flagged to @andrewroth 

The only reason why it could be returning as a string is, otherwise they would have to set up each field that returns as a suggested change in the graphQL schema.